app/views/shared/_repository_row_reminder.html.erb

```diff
@@ -9,6 +9,19 @@
           <%= t('repository_row.reminder.stock_low', stock_formated: reminder.value.formatted ).html_safe %>
         <% end %>
       </div>
+    <% elsif reminder.value_type == 'RepositoryDateTimeValueBase' %>
+      <div class="row-reminders-title"><strong><%= t('repository_row.reminder.date_reminder') %></strong></div>
+      <div class="row-reminders-body">
+        <%if reminder.value.data < Time.now.utc %>
+          <%= t('repository_row.reminder.item_expired') %>
+        <% else %>
+            <%= t('repository_row.reminder.date_expiration', date_expiration: 
+            pluralize(((reminder.value.data - Time.now.utc)/1.day).ceil, t('repository_row.reminder.day'))).html_safe %>
+        <% end %>
+      </div>
+      <% if reminder.repository_column.metadata["reminder_message"].present? %>
+        <div class="row-reminders-body" > <%= reminder.repository_column.metadata["reminder_message"] %> </div>
+      <% end %>
```

Comment:
Maybe you can make partial for each type of reminder, in the future when we will have more it will be harder.